### ISSUE TITLE
Kotlin Migration

### ISSUE THREAD

**Issue by faisalcodes** [AlfaazPlus/QuranApp#2]
As mentioned [here](https://github.com/AlfaazPlus/QuranApp/blob/master/CONTRIBUTING.md#before-you-start:~:text=Please%20note%20that%20all%20new%20code%20contributions%20must%20be%20written%20in%20Kotlin.), all the new codes must be written in Kotlin. We need to perform following actions to make the existing codes more efficient and keep them up to date -

- We should migrate all existing java codes to Koltin and traditional network operations to Kotlin coroutine.
- The migration should be done in an organized manner.
- Only a few files should be changed in a single PR. If the file is large, then one file at a time should be migrated.
- The files must **NOT** be converted using the android studio built-in tool. They must be written from scratch.
- Many files depend on several small and large files so we should start from smaller ones.
- The integrity of static fields and methods between java and Koltin codes should be maintained.

**Comment by ARBoyGo** [comment]
Yes, Need to make this app as kotlin, As it is easy for beginners also alot of people can contribute. The theming is also wonderful.

**Comment by Pantyhose-X** [comment]
to Go
